thirdparty/github.go

```diff
@@ -835,6 +836,31 @@ func GithubUserInOrganization(ctx context.Context, token, requiredOrganization,
 	return isMember, err
 }
 
+// AppAuthorizedForOrg returns true if the given app name exists in the org's installation list,
+// and has permission to write to pull requests. Returns an error if the app name exists but doesn't have permission.
+func AppAuthorizedForOrg(ctx context.Context, token, requiredOrganization, name string) (bool, error) {
+	httpClient := getGithubClientRetry(token, "AppAuthorizedForOrg")
+	defer utility.PutHTTPClient(httpClient)
+
+	client := github.NewClient(httpClient)
+	// We can assume no org will have more than 100 installations.
+	installations, _, err := client.Organizations.ListInstallations(ctx, requiredOrganization, &github.ListOptions{PerPage: 100})
```

Comment:
Should we put some kind of log to alert in the future if this hits 100+ installations?

Reply:
eh I guess I'd rather just use pagination handling than that